CenterForOpenScience/isp
# Add hebrew translation + RTL surveys

⚠️ Companion to, and must be merged after:
https://github.com/CenterForOpenScience/exp-addons/pull/215

Refs: 
https://openscience.atlassian.net/browse/LEI-344
https://openscience.atlassian.net/browse/LEI-348

## Purpose
Add hebrew translation.

Use this as a test case for integrating RTL survey layouts.

## Summary of changes
- Change survey layout logic
- Add hebrew

## TODO
- [x] RTL survey layouts
  - [x] Radio button order
  - Possibly card sort, but not unless the client asks

- [x] Need a good way to identify if locale is RTL; service doesn't expose automatically
See `https://github.com/jamesarosen/ember-i18n/issues/375`

## Testing notes
- Verify correct consent form shown for study ID `ISRA1.HE`.
- Verify that hebrew text is shown in the correct direction
- On the survey section, verify that the scale for any radio button question goes `1-->10` for english, and `10-->1` for hebrew
